### ISSUE TITLE
vscode 1.45.1 Windows 10 64 Bit. Extension not working (Tested on python)

### ISSUE THREAD

**Issue by mimansa-dev** [pilat/split-lines#20]
This extension does not work on the below indicated vscode version installed on Windows 10 as tested with python files.

Version: 1.45.1 (user setup)
Commit: 5763d909d5f12fe19f215cbfdd29a91c0fa9208a
Date: 2020-05-14T08:27:35.169Z
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Comment by jshnaidman** [comment]
It's also stopped working for me as well on Mac (with python)

Split Lines version 1.0.2

VS code Version: 1.45.1
Commit: 5763d909d5f12fe19f215cbfdd29a91c0fa9208a
Date: 2020-05-14T08:33:47.663Z (6 days ago)
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.4.0

**Comment by mimansa-dev** [comment]
Is there any other alternative that works with python files that you came across?

**Comment by jshnaidman** [comment]
> Is there any other alternative that works with python files that you came across?

Sorry for the late reply, I didn't get a notification of a reply. I haven't. I just split my own lines as of now and reply on autopep8 for the rest of my formatting.

**Comment by mimansa-dev** [comment]
I am splitting my lines manually but autopep8 joins them back despite the lines being more than then 120 characteres defined limit. It simply does not work. I think I will move to PyCharm shortly.

**Comment by sergioteula** [comment]
Same here, not working on Windows 10 since May. Any idea on how to solve it?

**Comment by vojtechmasa** [comment]
Is there a plan to fix this bug? It looks like this repo is not being maintained. It doesn't work for me with no language.

**Comment by huantianad** [comment]
Yeah, it sucks that this is broken. I haven't been able to find an alternative personally either.
